lib/_tls_wrap.js

```diff
@@ -633,6 +632,16 @@ TLSSocket.prototype._wrapHandle = function(wrap) {
   return res;
 };
 
+TLSSocket.prototype._wrapConnectedHandle = function(handle) {
```

Comment:
Shouldn't we use a symbol here rather than a underscode leading public property?
```suggestion
TLSSocket.prototype[kWrapConnectedHandle] = function(handle) {
```